apps/marketplace/components/Brief/BriefSpecialistResponseForm2.js

```diff
@@ -367,7 +368,7 @@ const BriefSpecialistResponseForm2 = ({
                       addOtherDocument()
                     }}
                   >
-                    Add another
+                    Add another{buttonAddedCount}
```

Comment:
This file shouldn't be changing as part of this work.  Can you please revert these changes?

Reply:
reverted codes to master